AdobeReader/AdobeReaderDC.download.recipe

```diff
@@ -3,67 +3,54 @@
 <plist version="1.0">
 <dict>
 	<key>Comment</key>
-	<string>For OS_Version, 10.14.0 or 10.15.0 are valid, not 10.14 or 10.15</string>
-    <key>Description</key>
-    <string>Downloads the latest Adobe Reader DC.</string>
+	<string>For OS_VERSION, 'Mac OS 10.14.0' or 'Mac OS 10.15.0' are valid, not 10.14 or 10.15</string>
+	<key>Description</key>
+	<string>Downloads the latest Adobe Acrobat Reader DC.</string>
 	<key>Identifier</key>
 	<string>com.github.autopkg.download.AdobeReaderDC</string>
 	<key>Input</key>
 	<dict>
-		<key>LANGUAGE</key>
-		<string>English</string>
-		<key>MAJOR_VERSION</key>
-		<string>AcrobatDC</string>
+		<key>NAME</key>
+		<string>AdobeReaderDC</string>
 		<key>OS_VERSION</key>
-		<string>10.15.0</string>
-	    <key>NAME</key>
-	    <string>AdobeReaderDC</string>
+		<string>Mac OS 10.14.0</string>
```

Comment:
Possible to preserve the `10.14.0` formatting style of this input variable? It would be highly desirable for people's existing overrides to continue working as expected.

Reply:
```
curl 'https://rdc.adobe.io/reader/products?os=10.14.0&api_key=dc-get-adobereader-cdn'
{"products":{"reader":[],"dcPro":[]},"addons":[]}%
```

It doesn't work as-is. A possible option would be to prepend `Mac OS` on to the front of it to preserve existing overrides, but I fear what issues that may cause in the future.

Reply:
I think prepending if needed is the right thing to ease compatibility. Perhaps we could output a `WARNING` to let people know they need to change their OS format.

Reply:
I'm thinking something like this:

```py
        os_version = self.env.get("os_version", OS_VERSION_DEFAULT)
        if not os_version.startswith("Mac OS"):
            self.output(
                "WARNING: Please update the OS_VERSION in your override "
                f"from '{os_version}' to 'Mac OS {os_version}'"
            )
            os_version = f"Mac OS {os_version}"
```

Reply:
Suggestion added and tested.